From c512e6dd3797b9706d5e7d9c20033ad3ac6a8bfc Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Mon, 18 Mar 2019 17:24:57 +0000
Subject: [PATCH] TOOLS-2101 jsstyle submodules too old for old perls Reviewed
 by: Cody Peter Mello <melloc@writev.io> Approved by: Cody Peter Mello
 <melloc@writev.io>

---
 .gitmodules  | 2 +-
 deps/jsstyle | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index 14e5f4b..6998f69 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,3 +1,3 @@
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = https://github.com/davepacheco/jsstyle.git
+	url = https://github.com/joyent/jsstyle.git
diff --git a/deps/jsstyle b/deps/jsstyle
index b9f5092..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit b9f50929ee2df140fd2d111f053b20593e068cdc
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
-- 
2.21.0

